src/openms/source/KERNEL/MSExperiment.cpp

```diff
@@ -503,7 +503,10 @@ namespace OpenMS
       }
       else
       {
-        String ms_run_location = path + "/" + filename;
+        // use Windows or UNIX path separator?
+        String actual_path = path.hasPrefix("file:///") ? path.substr(8) : path;
```

Comment:
I would probably prefer "/" as Qt does
https://doc.qt.io/qt-5/qdir.html#cleanPath

Reply:
I like the idea, but when I tried it just now, `QDir::cleanPath` didn't actually convert the backslashes - presumably because the "file:///" part makes it recognise the whole thing as a UNIX path. I could just go ahead and convert all "\" to "/", but I think technically "\" is allowed in UNIX paths.